src/modules/drawing/lib/ProfileChart.js

```diff
@@ -226,14 +228,17 @@ export default class ProfileChart {
             .attr('d', area)
             .attr('data-cy', 'profile-popup-area')
 
-        this.group
+        this.svg
```

Comment:
I recommend you use `this.group` here. I'll add some comments below on how to move the text from the group translation.